src/mappings/fspiopiso20022/quotes.ts

```diff
@@ -121,6 +125,27 @@ export const quotes_reverse = {
     "body.CdtTrfTxInf.ChrgsInf.Amt.ActiveOrHistoricCurrencyAndAmount": "body.payeeFspFee.amount",
     "body.CdtTrfTxInf.VrfctnOfTerms.IlpV4PrepPacket": "body.ilpPacket"
   }`,
+  put: `{
+    "$noDefaults": "true",
+    "body.GrpHdr.MsgId": { "$transform": "generateID" },
+    "body.GrpHdr.CreDtTm": { "$transform": "datetimeNow" },
+    "body.GrpHdr.NbOfTxs": { "$transform": "fixed", "value": "1" },
+    "body.GrpHdr.SttlmInf.SttlmMtd": { "$transform": "fixed", "value": "CLRG" },
+    "body.GrpHdr.PmtInstrXpryDtTm": "body.expiration",
+    "body.CdtTrfTxInf.PmtId.TxId": "params.ID",
+    "body.CdtTrfTxInf.Dbtr": "$context.isoPostQuote.CdtTrfTxInf.Dbtr",
+    "body.CdtTrfTxInf.DbtrAgt": "$context.isoPostQuote.CdtTrfTxInf.DbtrAgt",
+    "body.CdtTrfTxInf.Cdtr": "$context.isoPostQuote.CdtTrfTxInf.Cdtr",
+    "body.CdtTrfTxInf.CdtrAgt": "$context.isoPostQuote.CdtTrfTxInf.CdtrAgt",
+    "body.CdtTrfTxInf.ChrgBr": "$context.isoPostQuote.CdtTrfTxInf.ChrgBr",
+    "body.CdtTrfTxInf.IntrBkSttlmAmt.Ccy": "body.transferAmount.currency",
+    "body.CdtTrfTxInf.IntrBkSttlmAmt.ActiveCurrencyAndAmount": "body.transferAmount.amount",
+    "body.CdtTrfTxInf.InstdAmt.Ccy": "body.payeeReceiveAmount.currency",
+    "body.CdtTrfTxInf.InstdAmt.ActiveOrHistoricCurrencyAndAmount": "body.payeeReceiveAmount.amount",
+    "body.CdtTrfTxInf.ChrgsInf.Amt.Ccy": "body.payeeFspFee.currency",
+    "body.CdtTrfTxInf.ChrgsInf.Amt.ActiveOrHistoricCurrencyAndAmount": "body.payeeFspFee.amount",
+    "body.CdtTrfTxInf.VrfctnOfTerms.IlpV4PrepPacket": "body.ilpPacket"
+  }`,
```

Comment:
If this is the correct mapping for PUT /quotes, i.e this mapping now requires `$context.isoPostQuote` (the post quote message the put quote request is responding to) and no `headers` alternative, then we can remove `putTesting` mapping and update the expected test fixtures accordingly and the TypeGuard for the related facade function.

Reply:
We want both. There are cases where I want `putTesting` for a quick test without prior request context.
I've corrected `$context` to be optional on the type.